test/config/utility.h

```diff
@@ -83,6 +83,9 @@ class ConfigHelper {
   // Add the default SSL configuration.
   void addSslConfig();
 
+  // Renames the first listener to the name specified.
+  void renameListener(const char* name);
```

Comment:
Why `const char*`? I would go with either `absl::string_view` or `std::string` as this is C++.